mayastor/src/grpc/mod.rs

```diff
@@ -75,13 +83,15 @@ pub async fn sync_config<F, T>(future: F) -> GrpcResult<T>
 where
     F: Future<Output = GrpcResult<T>>,
 {
+    let guard = RECONFIGURING.lock().await;
```

Comment:
I might be mistaken but I thought that we already only had one gRPC call at a time?

Reply:
Maybe someone else can confirm?

Reply:
Maybe @gila or @jonathan-teh know?